Description: Adjusts the ivy configuration to work with ivy-debian-helper
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -166,7 +166,7 @@
 	</target>
 	
 	<target name="config-ivy" depends="ensure-ipp">
-		<ivy:configure file="buildScripts/ivysettings.xml" />
+		<ivy:configure file="/usr/share/ivy-debian-helper/ivysettings.xml" />
 	</target>
 	
 	<target name="deps" depends="ensureBuildDeps, ensureRuntimeDeps, ensureTestDeps" />
--- a/buildScripts/ivy.xml
+++ b/buildScripts/ivy.xml
@@ -10,7 +10,6 @@
 	<dependencies>
 		<dependency org="junit" name="junit" rev="4.13" conf="test -> default"/>
 		<dependency org="com.jcraft" name="jsch" rev="0.1.42" conf="build->default" />
-		<dependency org="projectlombok.org" name="jsch-ant-fixed" rev="0.1.45" conf="build" />
 		<dependency org="org.ow2.asm" name="asm" rev="9.1" conf="runtime, build -> default; contrib->sources" />
 		<dependency org="org.ow2.asm" name="asm-tree" rev="9.1" conf="runtime, build->default; contrib->sources" />
 		<dependency org="org.ow2.asm" name="asm-commons" rev="9.1" conf="runtime, build->default; contrib->sources" />
